book/guided-tour/README.md

```diff
@@ -991,7 +991,7 @@ tell *utop* to process the input, not to separate two declarations
     | Rect { lower_left; width; height } ->
       point.x    > lower_left.x && point.x < lower_left.x + width
       && point.y > lower_left.y && point.y < lower_left.y + height
-    | Segment { endpoint1; endpoint2 } -> false
```

Comment:
A catch-all case is not good style.  Changing this to `Segment _ -> false` would be better.

Reply:
updated.